package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurantpage",
+  "version": "1.0.0",
+  "description": "Restaurant Page with JS",
+  "private": true,
+  "scripts": {
+    "start:dev": "webpack-dev-server",
+    "build": "webpack",
+    "development": "webpack --env dev",
+    "start": "webpack-dev-server --open",
```

Comment:
we only one `start` here, kindly remove the one on line 7.
update before merging.